init.go

```diff
@@ -66,6 +66,7 @@ func Init(out io.Writer, nBitsForKeypair int) (*Config, error) {
 				"Access-Control-Allow-Methods": []string{"GET"},
 				"Access-Control-Allow-Headers": []string{"X-Requested-With", "Range"},
 			},
+			APICommands: DefaultGatewayAPICommands(),
```

Comment:
Will it end up writing these to the config? If it does, I'm worried we won't be able to add commands later without a migration.

Reply:
I guess alternatively we could keep the default an empty array?